packages/sync/test/SquashOperation.test.ts

```diff
@@ -143,4 +94,25 @@ describe("SquashOperations", () => {
     expect(opQueue.length).eqls(3);
 
   });
+
+  it("test noSquash Directive", () => {
```

Comment:
This is not how unit tests are build. They always tend to test single file. 
here we mixing 2 files in single test.  Actually this directive should be in squashing logic.  queue do not need to be polluted with the  business logic that is not queue oriented.